lib/optimist.rb

```diff
@@ -636,7 +708,10 @@ def parse(_paramlist, _neg_given)
   def type_format ; "" ; end
 
   def educate
-    (short? ? "-#{short}, " : "    ") + "--#{long}" + type_format + (flag? && default ? ", --no-#{long}" : "")
+    optionlist = []
```

Comment:
Noting that this is an undo of the short-argument padding/alignment.
If there can be multiple short args, then the padding doesnt make sense anymore or it would be a bit more complicated to align.